### DISCUSSION TITLE
License Change

### DISCUSSION THREAD

**Discussion by daveaglick** [statiqdev/Discussions#147] [General]
Hi everyone! As some of you may have noticed, the licenses are currently not linking correctly due to some changes on the License Zero site (see https://github.com/statiqdev/Statiq.Web/issues/992 and https://forum.artlessdevices.com/t/sic-transit-license-zero/291). As a result this has finally forced me to take a closer look at the licenses - something I've been meaning to do for a while. While I've made it clear I support free use by indie developers, bloggers serving ads, and other low-revenue users, that hasn't been explicit and it's been on my to-do list for a while to improve that and make it clear those users are excluded from needing to purchase a commercial license.

To that end I'm going to be relicensing Statiq effective immediately (or technically, with the next release I guess). The changes will only be more permissive, so no one is going to lose any access or be further restricted. In general, the impact will be an explicit public license exception for folks making **less than $100,000 USD and fewer than 10 employees**. This will be accomplished with a slightly modified [Polyform Small Business license](https://polyformproject.org/licenses/small-business/1.0.0/) instead of the current License Zero Prosperity public license. The private license you get when purchasing will remain the same License Zero Private license since I like that it's very similar to MIT with minimal but necessary changes to support the dual-licensing model.

- Why change anything?

I've wanted to make sure that very small business and individuals had open use rights. I'm not interested in charging a blogger who serves some ads on their site a license fee (though donations/sponsorship is always appreciated in those cases). I still love and support open source software and have many other OSS projects. I just think it's appropriate for a project of this scale to earn funding from business that can afford to fund it.

- Why change the Polyform Small Business from $1,000,000 and 100 employees to $100,000 and 10 employees?

The terms of Polyform Small Business doesn't take into account the requested license fee, and that's a big part of the context. If Statiq were being licensed for hundreds or thousands of dollars then the default exception numbers probably make sense. But Statiq is being licensed for a token amount - just enough to earn a little "go to the movies" money and make a point about funding. For that, a lower threshold seems appropriate to me.

- Will this change anything for me?

It shouldn't. The only folks it will impact are those who were considering licensing because of a strict interpretation of the original non-commercial terms of the License Zero Prosperity license and now won't be required to. I'm also still happy to discuss individual situations, but hopefully with the additional clarity of the new licenses that won't be required as much.

Happy to answer any additional questions anyone has.

**Comment by rdeago** [comment]
I won't discuss the legal aspects of the licenses, as I'm totally unqualified to. LGTM for what it's worth. The modified Small Business limits make perfect sense too.

I think the only piece missing is some form of guideline for contributors, plus probably a CLA for Statiq.Web and Statiq.Docs. If I were to contribute to those projects, I wouldn't mind you earning money from them (my contributions included), but if I were you I'd like this to be put in writing in clear legal terms and signed.

**Reply by daveaglick** [reply]
> I think the only piece missing is some form of guideline for contributors, plus probably a CLA for Statiq.Web and Statiq.Docs. If I were to contribute to those projects, I wouldn't mind you earning money from them (my contributions included), but if I were you I'd like this to be put in writing in clear legal terms and signed.

Huh, now that you mention it the CLA is gone! I actually used to have one rigged up and didn't notice it stopped working (the nice thing about integrations is that you don't have to think about them once they're set up - the bad thing is that once they're set up, it's easy not to notice when they break). I'll definitely be figuring that out again ASAP, and totally agree with the sentiment here. I love contributions, but also want to make sure everyone involves understands the expectations and intent.
